app/assets/stylesheets/modules/mfa.css

```diff
@@ -7,6 +7,29 @@
   flex: 1;
 }
 
+.mfa__header-wrapper {
+  display: flex;
+  align-items: center;
+  gap: 12px;
+}
+
+.t-body .mfa__header {
+  display: inline-block;
+  padding-top: 0px;
```

Comment:
I don't think these styles are needed?

Reply:
removing the `display: inline-block` makes the MFA LEVEL header a lil close too the "Multi-factor authentication" header, the `padding-top: 0px` was to override the default existing `.t-body h2` [padding-top of 30px](https://github.com/rubygems/rubygems.org/blob/222adb8f4078499ec3064dd8c51a390ca2e22a5d/app/assets/stylesheets/type.css#L149) (me and Betty agreed it was too much extra padding)

Reply:
<p align="center">
  <kbd>
    <img src="https://github.com/rubygems/rubygems.org/assets/20158582/1e918fec-0901-4087-b81d-df42b823e450" width=500>
    <img src="https://github.com/rubygems/rubygems.org/assets/20158582/a768bc01-0e15-492b-9702-0744f24b05b1" width=500>
  </kbd>
</p>

With / Without

Reply:
> We should keep the padding rule

Ah, I didn't notice when I was playing around with the styles, makes sense!

Reply:
One last thing 🙈, without the `display: inline-block` the padding of the "Multi-factor authentication" is no longer respected after a device (OTP or webauthn) is added.

<p align="center">
  <kbd>
    <img src="https://github.com/rubygems/rubygems.org/assets/20158582/04ed143a-b4cb-4e41-9cf5-1dc47db28b63" width=300>
    <img src="https://github.com/rubygems/rubygems.org/assets/20158582/6bec82f1-e8f0-4783-8898-2155da8e3afa" width=300>
  </kbd>
</p>

Reply:
Actually I take it back, the .mfa-levels block was missing the `<div class="mfa__header-wrapper">` the padding works as expected now :)